modules/nf-lineage/src/main/nextflow/lineage/model/TaskRun.groovy

```diff
@@ -64,20 +64,12 @@ class TaskRun implements LinSerializable {
      * Architecture defined in the Spack environment used for the task run
      */
     String architecture
-    /**
-     * Global variables defined in the task run
-     */
-    Map globalVars
```

Comment:
GlobalVars are used to calculate the hash. I think it shouldn't be removed

Reply:
I was going to remove because using global vars is an anti-pattern and will likely become an error under the strict syntax -- but now I realize it will still be useful with the `diff` command. So we should keep it for now and remove it in a future version where the task hash does not include global variables